columnflow/calibration/cms/jets.py

```diff
@@ -951,6 +957,11 @@ def jer(self: Calibrator, events: ak.Array, **kwargs) -> ak.Array:
     return events
 
 
+jer_horn_handling = jer.derive("jer_horn_handling", cls_dict={
```

Comment:
Could you add a reference (Twiki or else) where you took these values from?

Reply:
apparently, the recommended region is 2.5-3.0
I added the source and updated the Calibrator